data/aliases.js

```diff
@@ -236,20 +237,31 @@ exports.BattleAliases = {
 
 	// pokemon
 	"aboma": "Abomasnow",
+	"alaka": "Alakazam",
+	"ape": "Infernape",
 	"bliss": "Blissey",
 	"chomp": "Garchomp",
 	"cofag": "Cofagrigus",
+	"conk": "Conkeldurr",
+	"croak": "Toxicroak",
 	"cress": "Cresselia",
 	"dnite": "Dragonite",
 	"don": "Groudon",
 	"dogars": "Koffing",
+	"dos": "Gyarados",
```

Comment:
I've only seen dos used to reference gyarados. The only way I've seen zapdos abbreviated is as zap, which is pretty often.